index.php

```diff
@@ -0,0 +1,70 @@
+<?php
+$monsterArray = [];
+
+if (isset($_GET['id'])) {
+    $db = new PDO(
+        'mysql:host=192.168.20.20; dbname=collection',
+        'root',
+        ''
+    );
+
+    $db->setAttribute(
+        PDO::ATTR_DEFAULT_FETCH_MODE,
+        PDO::FETCH_ASSOC
+    );
+
+    $idToSearchFor = $_GET['id'];
+
+    $sql = $db->prepare(
+        'SELECT * FROM `monsters`;'
+    );
+
+    $sql->bindParam(
+        'id',
+        $idToSearchFor,
+        PDO::PARAM_INT
+    );
+
+    $sql->execute();
+
+    $monsterArray = $sql->fetchAll();
+}
+
+function displayMonster($Monsterstats) {
+    echo $Monsterstats [0]['type'];
+}
+
+?>
+
+<html lang="en">
+    <head>
+        <title>Monster collection</title>
+        <link rel ="stylesheet" type="text/css" href="index.css">
+        <meta name="viewport contents" content="device-width, initial-scale=1.0">
+    </head>
+    <body>
+        <header>
+            <div class="Hero-container">
+                <h1 class="Hero">Monster collection</h1>
+            </div>
+        </header>
+    <section class="monsters">
+        <?php
+        foreach($monsterArray as $item) {
+        echo '<div class="card-container">';
+            echo '<div class="card">';
+                echo '<div class="card-title">' . $item['type'] . '</div>';
+                echo '<div class="card-text">Element = '  . $item['element'] . '</div>';
+                echo '<div class="card-text">Healthpoints = '  . $item['healthpoints'] . '</div>';
+                echo '<div class="card-text">Strength = '  . $item['strength'] . '</div>';
```

Comment:
better to have all this echoing as a function making a string outside of the HTML that you then call on to keep everything neater.